portfolio/src/main/java/com/google/sps/data/LoginResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.google.sps.data;
+
+public class LoginResponse {
+  private boolean isLoggedIn;
+  private String url;
+
+  public void setLoggedIn(boolean isLoggedIn) {
```

Comment:
this looks fine, but if you are interested, you can try to read these:
- protobuf
- builder pattern

Reply:
tl;dr for builder pattern
if you change `public void setLoggedIn` to:
```
public LoginResponse setLoggedIn(boolean isLoggedIn) {
    this.isLoggedIn = isLoggedIn;
    return this
}
```
you will be able to do:
```
responseObj.setLoggedIn(true).setUrl(url)
```

Reply:
Ok will read up on those, thank you!